opentelemetry-api/src/opentelemetry/metrics/examples/pre_aggregated.py

```diff
@@ -13,24 +13,20 @@
 # limitations under the License.
 
 # pylint: skip-file
-from opentelemetry.metrics import LabelKey, LabelValue, Meter
+from opentelemetry import metrics
 
-METER = Meter()
-LABEL_KEYS = [
-    LabelKey("environment", "the environment the application is running in")
-]
-COUNTER = METER.create_int_counter(
-    "sum numbers",  # pragma: no cover
+METER = metrics.Meter()
+COUNTER = METER.create_counter(
+    "sum numbers",
     "sum numbers over time",
     "number",
-    LABEL_KEYS,
+    metrics.ValueType.FLOAT,
```

Comment:
will this also support just passing the object in? like `float`?

Reply:
I understand idiomatic python as using base object types where applicable, so I feel like that's a good strategy.